Shared/Samples/Show realistic light and shadows/ShowRealisticLightAndShadowsView.swift

```diff
@@ -59,13 +59,11 @@ struct ShowRealisticLightAndShadowsView: View {
         .toolbar {
             ToolbarItemGroup(placement: .bottomBar) {
                 Text(dateTimeText)
-                Button("Mode") {
-                    isShowingSettings = true
-                }
-                .confirmationDialog("Choose a lighting mode for the scene view.", isPresented: $isShowingSettings, titleVisibility: .visible) {
-                    ForEach(SceneView.SunLighting.allCases, id: \.self) { mode in
-                        Button(mode.label) {
-                            lightingMode = mode
+                
+                Menu("Mode") {
```

Comment:
![iShot_2023-04-11_13 44 19](https://user-images.githubusercontent.com/9660181/231286146-eb2c1700-8212-46d5-822c-0c049d25c350.png)

I tried this sample on the smallest screen device we currently support and the text clips, so I tends to keep the button label as "Mode". Will wait for Nick's thoughts.

Reply:
It was both actually. The action sheet seemed wrong, but if one uses a picker then it felt it would be good to see what the current mode was.

Perhaps it can go on a subsequent row. The spacing of the time slider and time label is pretty big - I bet it could be tidied up a bit more and not make the entire panel any/much taller.

Reply:
OK. I'll try to move the time label to the top status label, and see how it looks.